src/transformers/utils/import_utils.py

```diff
@@ -840,15 +840,13 @@ def is_torch_xpu_available(check_device=False):
     return hasattr(torch, "xpu") and torch.xpu.is_available()
 
 
+@lru_cache()
 def is_bitsandbytes_available():
-    if not is_torch_available():
-        return False
+    from transformers.integrations.integration_utils import validate_bnb_backend_availability
 
-    # bitsandbytes throws an error if cuda is not available
-    # let's avoid that by adding a simple check
-    import torch
-
-    return _bitsandbytes_available and torch.cuda.is_available()
+    return (
+        _bitsandbytes_available and is_torch_available() and validate_bnb_backend_availability(raise_exception=False)
```

Comment:
I think importing `bitsandbytes` would fail if Torch isn't available, so the check for Torch is likely moot?
```suggestion
        _bitsandbytes_available and validate_bnb_backend_availability(raise_exception=False)
```

Reply:
I'd like to keep the original logic to check torch, it is not important because bnb cannot run without torch.

Reply:
Erm, why? It's a tiny thing, but why make the computer do this tiny bit of extra work for no good reason?

Reply:
You were right, I have removed the torch check.